common/djangoapps/util/markup.py

```diff
@@ -0,0 +1,51 @@
+"""
+Utilities for use in Mako markup.
+"""
+
+from django.utils.translation import ugettext as django_ugettext
+from django.utils.translation import ungettext as django_ungettext
+import markupsafe
+
+
+# So that we can use escape() imported from here.
+escape = markupsafe.escape                      # pylint: disable=invalid-name
+
+
+def ugettext(text):
+    """Translate a string, and escape it as plain text.
+
+    Use like this in Mako::
```

Comment:
@cahrens I think the "::" here and below starts a literal block to include code samples.